### PR TITLE
Online filter

### PR DESCRIPTION
**Pull request by taigercub**
Add functionality to Online Only filter

### CODE REVIEW

File: package.json
Comment (Peyton3995) [inline package.json:1]: Please do not include anything regarding package.json in your PR.

**Review by Peyton3995 — CHANGES_REQUESTED** [review]

File: package.json
Comment (taigercub) [reply package.json:1]: fixed

**Review by Peyton3995 — APPROVED** [review]
Pull this down and it works. Good work

**Comment by Peyton3995** [comment]
> Pull this down and it works. Good work

**Review by Peyton3995 — APPROVED** [review]
This actually adds your changes to the git-ignore. This is fine since it will not actually change anything and gives us better coverage.

**Review by owenmock22 — APPROVED** [review]
Looks good

**Pull request merged**